test/test_facebook.py

```diff
@@ -151,14 +154,22 @@ def test_invalid_format(self):
 
 class TestAuthURL(FacebookTestCase):
     def test_auth_url(self):
+        default_version = facebook.VALID_API_VERSIONS[0]
+        api = facebook.GraphAPI(version=default_version)
         perms = ['email', 'birthday']
         redirect_url = 'https://localhost/facebook/callback/'
 
-        expected_url = facebook.FACEBOOK_OAUTH_DIALOG_URL + urlencode(
-            dict(client_id=self.app_id,
-                 redirect_uri=redirect_url,
-                 scope=','.join(perms)))
-        actual_url = facebook.auth_url(self.app_id, redirect_url, perms=perms)
+        expected_url = ''.join([
+            facebook.FACEBOOK_URL,
+            'v' + default_version + '/',
+            facebook.FACEBOOK_OAUTH_DIALOG_URI,
+            urlencode(
+                dict(client_id=self.app_id,
+                     redirect_uri=redirect_url,
+                     scope=','.join(perms)))
+        ])
```

Comment:
I think this should use `.format` instead of `.join` and `api.version` instead of `default_version` (which doesn't need to exist for reasons I outlined earlier).